src/queuemailerl_event.erl

```diff
@@ -110,10 +110,26 @@ build_error_mail(#event{error = #error{to = To, subject = Subject,
 
 %% Internal
 
+trim_binary(Bin, Junks)
+  when is_list(Junks) ->
+    lists:foldl(fun (Bin0, Junk) ->
+                        trim_binary(Bin0, Junk)
+                end, Bin, Junks);
+trim_binary(Bin, Junk) ->
```

Comment:
Feels like an rather complex way of stripping whitespaces.
trim_binary(<<" ", Rest>>) -> trim_binary(Rest);
trim_binary(Done) -> Done.